Applications/DataExplorer/mainwindow.cpp

```diff
@@ -1096,6 +1096,16 @@ void MainWindow::HideWindow()
 	this->hide();
 }
 
+void MainWindow::LoadFile(const QString &fileName)
+{
+	ImportFileType::type t;
+	QString ext = QFileInfo(fileName).suffix();
+	if (ext=="msh" || ext=="vtu" || ext=="gli" || ext=="gml") {
+		t = ImportFileType::OGS;
+		this->loadFile(t,fileName);
```

Comment:
you can completely remove the ```t``` variable and just write
```this->loadFile(ImportFileType::OGS, fileName);``` 
this saves 2 lines of code .... 3 if you also remove the braces ...